crates/storage-query-datafusion/src/node/schema.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+#![allow(dead_code)]
+
+use datafusion::arrow::datatypes::DataType;
+
+use crate::table_macro::*;
+
+define_table!(
+    /// Introspect cluster nodes.
+    node(
```

Comment:
Assuming this is the table name, can this be plural?

Reply:
This is not the table name actually this define the builder types (Build, Row, etc...) as generated by the macro. I tried to follow the convienction used across the other table types (sys_deployment, and sys_service, for example). 

I also noticed that the table names follow the same singular pattern. (e.g sys_deployment) and I think it makes sense to keep the table name in a singular form.

Reply:
Okay, while this here will remain in singular form because it make sense for the types to be in a singular form (e.g. `NodeRowBuilder`) the table name itself has changed to `nodes` [below](https://github.com/restatedev/restate/pull/2913/commits/617cce97752aa3b94f5dda03303a9e05ee32ce44#diff-e20f51133333a50fc916de33dcb2c8d03e0a85b748b92a7d341cb1f830745d50R33)